docs/governance/community-guidelines/releases/cmb.md

```diff
@@ -0,0 +1,106 @@
+
+A release cycle shall be a minimum of one month, during which time a Release Manager will solicit and arbitrate feedback from the reviewers prior to approving and initiating the release.
+
+### Release Manager Responsibilities
+
+The release manager is not a unilateral authority on the release, rather they are the representative of the group's opinions. Insomuch as they represent the CMB, the release manager holds the final guidance in the lifecycle of an asset.
+
+The release manager will be responsible for the following:
+
+- Collaborate with the CCC working group leads to ensure that the asset is ready for review.
+- Issue an announcement to the CMB, containing:
+  - Links to the asset under review
+  - Desired release date
+  - Deadline for initial responses (two weeks prior to desired release date)
+  - Instructions for participating in this review cycle
+- When a change request (CR) is received:
+  - Evaluate the quality of the CR. If necessary, request adjustments for clarity or conciseness.
+  - Relay the CR to all participating reviewers
+  - At least two members must agree on a CR before it moves forward, with majority opinion ruling when there is dissent.
+  - When discussion has been stabilized for at least 48 hours, determine the status of the CR
+  - If the CR is affirmed by the CMB, create a GitHub issue detailing the CR. Tag and notify the appropriate working group.
+  - If the CR is not affirmed by the CMB, notify the change requestor. The CR should not be resubmitted unless there are substantial changes to the request.
+- When all outstanding requests have been resolved and requested changes have been applied, initiate the release.
+  - Ensure that the release is no sooner than the expected delivery date, and that all actions follow the current processes of the [Delivery WG].
+
+### Reviewer Responsibilities
+
+Members are **not** obligated to review every release but will be notified and may choose to engage in reviews.
+
+When engaging, the following is expected of a CMB member:
+
+- Be thorough, thoughtful, and provide detailed feedback before requesting changes.
+  - Gather feedback from colleagues as needed to support a review.
+- If changes are requested, communicate clearly and promptly through the channels outlined by the Release Manager for the current release cycle.
+  - When a change request (CR) is received, the Release Manager will open discussions and facilitate responses from the board.
+- Members are encouraged to respond within 7 days if they have input on a CR.
+  - The Release Manager logs any dissenting opinions and communicates the majority decision.
+- A release cannot proceed without 5 approvals; members are encouraged to help meet this threshold by approving or requesting changes.
+
+### Qualifications for Participation
+
+Individuals of any background or experience level may participate in a review.
+
+To approve or request changes, an individual must be an appointed CMB member in good standing.
+
```

Comment:
I think it would make sense to capture the CMB definitions that lie within this document into its own separate charter doc and refer to that one.

Reply:
Good shout. Let's remember to add that in a follow-up